library/worldgen/biome/src/main/java/org/quiltmc/qsl/worldgen/biome/mixin/TheEndBiomeSourceMixin.java

```diff
@@ -67,12 +67,10 @@ public abstract class TheEndBiomeSourceMixin extends BiomeSource {
 	 */
 	@Inject(method = "<clinit>", at = @At("TAIL"))
 	private static void modifyCodec(CallbackInfo ci) {
-		CODEC = RecordCodecBuilder.create((instance) -> {
-			return instance.group(RegistryOps.retrieveGetter(RegistryKeys.BIOME)).apply(instance, instance.stable(TheEndBiomeSource::m_biyltupg));
-		});
+		CODEC = RecordCodecBuilder.create((instance) -> instance.group(RegistryOps.retrieveGetter(RegistryKeys.BIOME)).apply(instance, instance.stable(TheEndBiomeSource::method_46680)));
```

Comment:
I think the {} was intentional as a way of shortening the line length? I won't ask for changes here though

Reply:
...because I'd rather do the changes myself :P